### DISCUSSION TITLE
Zoom in theme

### DISCUSSION THREAD

**Discussion by lknelsys** [quasarframework/quasar#16898] [Ideas / Proposals]
All of the quasar's components are too large and make the system too coarse.

As the sizes are generally proportional, I tried to reduce the screens with a simple css:

```
body>*:not(.fullscreen, [id^="q-portal"]),
.fullscreen>* {
  zoom: var(--zoom-level);
}

[id^="q-portal"] .q-list {
  zoom: var(--zoom-level);
}
```

This actually worked and made the system much better visually, but elements that have their positioning set via javascript such as menus, tooltips, among others, started to be out of position.

It would be nice if it already had a scaling feature so we could simply increase or decrease it without needing any workarounds.

**Comment by aryankarim** [comment]
Needing this feature currently. All my popups and proxies are out of position. Though with browser zoom everything get positioned correctly. I'm guessing there might be a function we could call at least to propagate the new position to all the relative positions.

Might update if i find anything.
